mlir/test/Dialect/LLVMIR/roundtrip.mlir

```diff
@@ -750,6 +750,16 @@ llvm.func @experimental_noalias_scope_decl() {
   llvm.return
 }
 
+#alias_scope_domain2 = #llvm.alias_scope_domain<id = "domainid", description = "The domain">
+#alias_scope2 = #llvm.alias_scope<id = "stringid", domain = #alias_scope_domain2, description = "The domain">
+
+// CHECK-LABEL: @experimental_noalias_scope_decl
```

Comment:
```suggestion
// CHECK-LABEL: @experimental_noalias_scope_decl2
```
nit: I would probably rename to `@experimental_noalias_scope_decl_with_string_id` rather than postfixing with a number. In any case the check line should match the function name.